setup.py

```diff
@@ -30,16 +30,14 @@
 
 from __future__ import absolute_import
 
-import sys
 import re
 import os
 import platform
-from setuptools import setup, find_packages
 
-version = getattr(sys, "version_info", (0,))
+import ez_setup
+ez_setup.use_setuptools('20.3')
 
```

Comment:
No. If you do that, the invocation of setup() will fail due to the requirements parse error before setuptools has a chance to update itself.